utils/check_modules.sh

```diff
@@ -41,8 +41,8 @@ set -e
 if [ ! -d "scons_local" ];then
   cd ..
 fi
-PYTHONPATH=$PWD/utils:$PWD/src/tests/ftest/util/
-PYTHONPATH=$PYTHONPATH:$PWD/src/tests/ftest/util/apricot/
+PYTHONPATH=$PWD/utils:$PWD/install/lib/daos/TESTING/ftest/util/
```

Comment:
should this be lib64 too or is it lib for python?

Reply:
This should be lib, util for ftest, not libraries (that go to lib64).

Reply:
I am still on the fence about this one and so it's subject to change.

I don't believe there is actually anything 64-bit specific in there so lib64/daos just seems wrong.  But it's also pretty clear from RPM packaging macros that on 64-bit machines, they don't want you using /usr/lib for anything.

So it probably needs some more investigation as to what best/accepted practices are.